src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,13 +28,21 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        
+        this.sequenceNumber = nextSequenceNumber;
+        Person.nextSequenceNumber += 1;
```

Comment:
No need to use `Person.` here. Also, you can use shorter form `nextSequenceNumber++`.